teps/0023-implicit-mapping.md

```diff
@@ -6,6 +6,8 @@ authors:
 creation-date: 2020-10-01
 last-updated: 2021-12-15
 status: implemented
+superseded-by:
+- TEP-0104
```

Comment:
For reviewers, [TEP metadata](https://github.com/tektoncd/community/blob/main/teps/0001-tekton-enhancement-proposal-process.md#tep-metadata) as described in TEP-0001

Reply:
This will need to be switched to `TEP-0107`. =)

Reply:
updated, thanks :)